ACMILabs/acmi-api
# #30 Add TSV exporter

Resolves #30

Add a TSV exporter.

### Acceptance Criteria
- [x] Add a TSV exporter that exports all Works API data to `app/tsv/works.tsv`

### Relevant design files
* None

### Testing instructions
1. Start your dev environment: `docker-compose -f docker-compose-base.yml up`
1. Run: `docker exec -it api python` and `import app.api as a` and `x = a.XOSAPI()` and `x.generate_tsv('works')`
1. Open the TSV `app/tsv/works.tsv` and see that the data looks correct

### DoD
For requester to complete:
